config/gulp/file-upload.js

```diff
@@ -44,7 +46,9 @@ function uploadFile() {
   });
 
   return gulp
-    .src("content/uploads/_working-files/to-process/*")
+    .src("content/uploads/_working-files/to-process/*", { 
+      removeBOM: true, 
```

Comment:
> [!NOTE]  
> We'll have to test slides and/or spreadsheet for this.

Reply:
If you log in to S3 you will see the PowerPoints I was using to test. Have not deleted them yet. 

I tested personally, 

-  image and slideshow, 
-  just slideshow
-  just image.